CIScripts/BuildFunctions.ps1

```diff
@@ -69,61 +73,73 @@ function Invoke-DockerDriverBuild {
            [Parameter(Mandatory = $true)] [string] $CertPath,
            [Parameter(Mandatory = $true)] [string] $CertPasswordFilePath)
     
+    $TimeLogger.LogAndPush("Docker driver build")
     $Env:GOPATH=pwd
+    $srcPath = "$Env:GOPATH/src/$DriverSrcPath"
     
     New-Item -ItemType Directory ./bin
     Push-Location bin
 
-    Write-Host "Installing test runner"
-    go get -u -v github.com/onsi/ginkgo/ginkgo
-
-    Write-Host "Building driver"
-    go build -v $DriverSrcPath
+    $TimeLogger.LogAndMeasure("Installing test runner", {
+        go get -u -v github.com/onsi/ginkgo/ginkgo
+    })
 
-    $srcPath = "$Env:GOPATH/src/$DriverSrcPath"
-    Write-Host $srcPath
+    $TimeLogger.LogAndMeasure("Building driver", {
+        go build -v $DriverSrcPath
+        Write-Host $srcPath
```

Comment:
Is this `Write-Host` necessary here? If so, could you move it to the place where `$srcPath` is assigned.